src/main/java/kitchenpos/table/application/CreateOrderTableValidationHandler.java

```diff
@@ -0,0 +1,27 @@
+package kitchenpos.table.application;
+
+import java.util.NoSuchElementException;
+import kitchenpos.order.event.CreateOrderEvent;
+import kitchenpos.table.domain.OrderTable;
+import kitchenpos.table.domain.OrderTableRepository;
+import org.springframework.context.event.EventListener;
+import org.springframework.stereotype.Component;
+
+@Component
+public class CreateOrderTableValidationHandler {
+
+    private final OrderTableRepository orderTableRepository;
+
+    public CreateOrderTableValidationHandler(OrderTableRepository orderTableRepository) {
+        this.orderTableRepository = orderTableRepository;
+    }
+
+    @EventListener(CreateOrderEvent.class)
```

Comment:
```suggestion
    @EventListener
```

매개변수로 타입을 알 수 있기 때문에 매번 추가하지 않으셔도 됩니다 ㅎㅎ

Reply:
수정하겠습니다!